src/components/RepoListItem.js

```diff
@@ -1,32 +1,34 @@
 import React from "react";
-import Label from "../styles/Label";
 import {FlexStart, FloatRight, FloatLeft, FlexColumn, FlexHeader, FlexCenter} from "../styles/Grid";
 import Octicon, {getIconByName} from "@primer/octicons-react";
 import Avatar from "../styles/Avatar";
 import {chevronRight} from "../icons";
+import {humanizeNumber} from "../lib/humanizeNumber";
 
 function RepoListItem({goal, stars}) {
   return (
     <FlexHeader>
       <FloatLeft>
         <FlexCenter>
-          <Avatar alt="avatar" src={`https://avatars.githubusercontent.com/${goal.title.split("/")[0].replace(/\s+/g, "")}`} />
+          <Avatar
+            alt="avatar"
+            src={`https://avatars.githubusercontent.com/${goal.full_name.split("/")[0].replace(/\s+/g, "")}`}
```

Comment:
the base URL for https://avatars.githubusercontent.com could be moved to a constant/global variable

Reply:
This could also fetch an image that is equivalent to the 2.5rems, not the full-quality image for improved performance